### DISCUSSION TITLE
How does Celeste compare to Nextcloud Desktop desktop if I only want to sync with my Nextcloud server?

### DISCUSSION THREAD

**Discussion by sylvesterroos** [hwittenborn/celeste#172] [General]
In other words, why might I choose this project over Nextcloud Desktop as a sync client?

**Comment by Fyafir** [comment]
The project is great, it opens up a range not only for NextCloud, but also for Google Drive, which currently handles synchronization best is InSync, but I'm sure this project will be much better and simpler.
